src/CommunityToolkit/Xamarin.CommunityToolkit/Startup/ToolkitPlatform.android.cs

```diff
@@ -1,26 +1,24 @@
 ﻿using Android.Content;
+using Xamarin.Forms.Platform.Android;
 
 namespace Xamarin.CommunityToolkit
 {
 	/// <summary>
 	/// Platform extension methods.
 	/// </summary>
-	public static partial class ToolkitPlatform
+	static class ToolkitPlatform
 	{
 		/// <summary>
 		/// Gets the <see cref="Context"/>.
 		/// </summary>
-		internal static Context Context { get; private set; }
-
-		/// <summary>
-		/// Initializes the Android <see cref="Context"/>.
-		/// </summary>
-		/// <param name="context">
-		/// The current <see cref="Context"/>.
-		/// </param>
-		public static void Init(Context context)
+		public static Context Context
 		{
-			Context = context;
+			get
+			{
+				var page = Forms.Application.Current.MainPage;
```

Comment:
Not sure if makes sense to add a `private field` here to cache the `Context`. XF works with just one context so makes sense to cache it but could be a scenario that exists more than one (?)

Reply:
I thought about this and decided against it because I didn't want the `context` being held in memory by this static class. That means the Garbage Collector would never destroy this. While reading through many Android forums and best practices they consider it a practice to avoid in stored the current context.

Sometimes in Xamarin.Forms we don't have a choice due to technical limitations. I think this solution is a great compromise and it doesn't appear to be an expensive call to retrieve the `context`

Reply:
So let's use this way, in the future if this introduces any issue we think in a solution for it.